### DISCUSSION TITLE
How to hide the SNPCoverage view in AlignmentsTrack ?

### DISCUSSION THREAD

**Discussion by tao-bioinfo** [GMOD/jbrowse-components#3540] [General] [answered]
When I open an `AlignmentsTrack` (e.g. BAM), it automatically shows a SNPCoverage view in the top. I want to hide the SNPCoverage view by default. How should I do? Thank you!

**Comment by tao-bioinfo** [comment, marked as answer]
Oh, it seems using "Pileup Display" is enough

**Reply by cmdcolin** [reply]
super. if you wanted to load your data using pileup display by default in the config file, can do something like this (configure the default display to be the pileup display)

```
{
      "type": "AlignmentsTrack",
      "trackId": "mytrack",
      "name": "mytrack",
      "assemblyNames": ["volvox"],
      "adapter": {
        "type": "CramAdapter",
        "cramLocation": {
          "uri": "volvox-sorted-altname.cram"
        },
        "craiLocation": {
          "uri": "volvox-sorted-altname.cram.crai"
        },
        "sequenceAdapter": {
          "type": "TwoBitAdapter",
          "twoBitLocation": {
            "uri": "volvox.2bit"
          }
        }
      },
      "displays": [
        {
          "type": "LinearPileupDisplay",
          "displayId": "mytrack_pileupdisplay"
        }
      ]
    },
```

this is done for this track in our sample config https://jbrowse.org/code/jb2/v2.4.0/?config=test_data%2Fvolvox%2Fconfig.json&session=share-SpV4naxA8d&password=sPcPA

**Reply by cmdcolin** [reply]
Also note: If your data is displaying e.g. a genome->genome alignment, you could use the SyntenyTrack which has the LGVSyntenyDisplay...this is an interesting track type that uses the PileupDisplay by default but is geared towards synteny data e.g. in PAF format https://jbrowse.org/code/jb2/v2.4.0/?config=test_data%2Fvolvox%2Fconfig.json&session=share-cg0rm_X%2F81&password=nfqme

that config looks like

```

    {
      "type": "SyntenyTrack",
      "trackId": "volvox_inv_indels",
      "name": "volvox_inv_indels",
      "category": ["Synteny"],
      "adapter": {
        "type": "PAFAdapter",
        "pafLocation": {
          "uri": "volvox_inv_indels.paf",
          "locationType": "UriLocation"
        },
        "assemblyNames": ["volvox_random_inv", "volvox"]
      },
      "assemblyNames": ["volvox_random_inv", "volvox"]
    }
```
